koku/cost_models/sql_parameters.py

```diff
@@ -93,5 +102,4 @@ def build_tag_based_rate_query(self, tag_based_price_list, metric_name):
                 }:
                     sql_params["value_rates"] = value_rates
                 tag_parameters.append(sql_params)
-        sql_file = pkgutil.get_data("masu.database", "trino_sql/openshift/cost_model/hourly_cost_vm_tag_based.sql")
-        return sql_file.decode("utf-8"), tag_parameters
+        return tag_parameters
```

Comment:
I'm not a fan of the inconsistency this creates with `build_vm_count_hourly_query`. Maybe we should update `build_vm_count_hourly_query` to only build and return the params so that it is consistent with what this func returns?

Reply:
For me, the differences in the naming structure for the two methods were enough to highlight the difference in the logic. One builds a query; while the other just builds the parameters. However, I can also see your point, I wouldn't be opposed to changing the name of the hourly and having it just return parameters as well.